analyzers/rspec/cs/S5547_c#.html

```diff
@@ -1,24 +1,38 @@
 <p><a href="https://en.wikipedia.org/wiki/Strong_cryptography">Strong cipher algorithms</a> are cryptographic systems resistant to cryptanalysis, they
-are not vulnerable to well-known attacks like brute force attacks for example. </p>
-<p>It is recommended to use only cipher algorithms intensively tested and promoted by the cryptographic community.</p>
+are not vulnerable to well-known attacks like brute force attacks for example.</p>
```

Comment:
Please revert the S5547_* changes - we have #3330 to fix first (we'll deal with it in the upcoming sprint)

We've been reverting them manually in the past releases as well (see notes in #3559 and #3607)